### PR TITLE
Add new rules

### PR DESCRIPTION
**Pull request by imwatsi**
I just noticed that in my changes, I had left out the heading of the new rules for Blog Posts :)

### CODE REVIEW

**Comment by imwatsi** [comment]
Added the new rules also

File: index.html [hunk 1]
```diff
@@ -519,6 +519,7 @@ <h3 class="text-center">Sub-Project Rules</h3>
                             <div class="carousel-item">
                                 <h3 class="text-center">Development Rules</h3>
                                 <ul class="list-group">
+                                	<li class="list-group-item"><i class="fa fa-exclamation"></i> Only merged Pull Requests will be accepted or Open Source projects you maintain never posted on Utopian before, forks included as long as the fork is not just a mirror of the original one. Proof of work required.</li>
```
Comment (mktcode) [inline index.html]: You might call it pedantic but I can't accept this tab instead of single whitespaces as indentation. :P

**Comment by imwatsi** [comment]
Tabs removed :)

File: index.html (hunk 1 above)
Comment (imwatsi) [reply index.html]: I had not noticed.
Completely understood... I'm a perfectionist myself too :)

**Pull request merged**